### PR TITLE
Add page for UMH lite

### PR DESCRIPTION
**Pull request by bhashkarsharma**
# Description

- Fixes https://github.com/united-manufacturing-hub/MgmtIssues/issues/1778

Screenshot 2024-04-30 at 11 28 19

## Checklist

*You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [contributing guidelines](https://umh.docs.umh.app/docs/development/contribute/new-content/add-documentation/) and the [code of conduct](https://github.com/united-manufacturing-hub/umh.docs.umh.app/blob/main/.github/CODE_OF_CONDUCT.md).
- [x] I have followed the [documentation](https://umh.docs.umh.app/docs/development/contribute/documentation/style/) style.

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+| Analytics                 |    ✓    |      |
+| Data Visualization        |    ✓    |      |
+| UNS                       |    ✓    |      |
+| Kafka and MQTT            |    ✓    |      |
+| Alerting                  |    ✓    |      |
+| Connectivity: OPC UA      |    ✓    |  ✓   |
+| Connectivity: Node-RED    |    ✓    |      |
+| Network Monitoring        |    ✓    |  ✓   |
+| UMH Data Model v1         |    ✓    |  ✓   |
+| Tag Browser for your UNF  |    ✓    |      |
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md:28]: UNS probably right?

File: umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+If you want the full-featured UMH experience, we recommend you to install the Classic version. You can see a comparison of the features between the two versions below.
+
+### UMH Classic vs UMH Lite
+
+| Feature                   | Classic | Lite |
+|---------------------------|:-------:|:----:|
+| Historian                 |    ✓    |      |
+| Analytics                 |    ✓    |      |
+| Data Visualization        |    ✓    |      |
+| UNS                       |    ✓    |      |
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md:21]: UNS = Kafka and MQTT

File: umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+description: |
+    Introducing UMH Lite - a new type of UMH installation.
+weight: 5000
+---
+
+If you are already using Unified Namespace, or have a Kafka / MQTT broker; and want to try out the basic UMH features, you can go for the UMH Lite installation.
+
+If you want the full-featured UMH experience, we recommend you to install the Classic version. You can see a comparison of the features between the two versions below.
+
+### UMH Classic vs UMH Lite
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md:14]: You can use these feature names here: 
Screenshot 2024-04-30 at 10 16 12

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md (hunk 3 above)
Comment (bhashkarsharma) [reply umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md:15]: Unfortunately markdown does not supported nested tables. Will see what I can do.

Comment (bhashkarsharma) [reply umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md:13]: Updated with raw HTML.

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**